stellarphot/io/tess.py

```diff
@@ -283,62 +252,104 @@ def __init__(self, tic_id, toi_table=DEFAULT_TABLE_LOCATION, allow_download=True
 
     @property
     def tess_mag(self):
+        """
+        tess_mag: float
+            The TESS magnitude of the target.
+        """
         return self._toi_table['TESS Mag'][0]
 
     @property
     def tess_mag_error(self):
+        """
+        tess_mag_error: float
+            The uncertainty in the TESS magnitude.
+        """
         return self._toi_table['TESS Mag err'][0]
 
     @property
     def depth(self):
         """
-        Depth, parts per thousand.
+        depth: float
+            The transit depth of the target in parts per thousand.
         """
         return self._toi_table['Depth (ppm)'][0] / 1000
 
     @property
     def depth_error(self):
         """
-        Error in depth, parts per thousand.
+        depth_error: float
+            The uncertainty in the transit depth in parts per thousand.
         """
         return self._toi_table['Depth (ppm) err'][0] / 1000
 
     @property
     def epoch(self):
+        """
+        epoch: float
+            The epoch of the transit.
+        """
         return Time(self._toi_table['Epoch (BJD)'][0], scale='tdb', format='jd')
 
     @property
     def epoch_error(self):
+        """
+        epoch_error: float
+            The uncertainty in the epoch of the transit.
+        """
         return self._toi_table['Epoch (BJD) err'][0] * u.day
 
     @property
     def period(self):
+        """
+        period: float
+            The period of the transit.
+        """
         return self._toi_table['Period (days)'][0] * u.day
 
     @property
     def period_error(self):
+        """
+        period_error: float
+            The uncertainty in the period of the transit.
+        """
         return self._toi_table['Period (days) err'][0] * u.day
 
     @property
     def duration(self):
+        """
+        duration: float
+            The duration of the transit.
+        """
         return self._toi_table['Duration (hours)'][0] * u.hour
 
     @property
     def duration_error(self):
+        """
+        duration_error: float
+            The uncertainty in the duration of the transit.
+        """
         return self._toi_table['Duration (hours) err'][0] * u.hour
 
     @property
     def coord(self):
+        """
+        coord: `astropy.coordinates.SkyCoord`
+            The coordinates of the target.
+        """
         return SkyCoord(ra=self._tic_info['ra'][0], dec=self._tic_info['dec'][0], unit='degree')
 
     @property
     def tic_id(self):
+        """
+        tic_id: int
+            The TIC ID of the target.
+        """
         return self._tic_info['ID'][0]
 
 @dataclass
 class TessTargetFile:
     """
-    A class to hold information about a TESS target file.
+    A data class to hold information about a TESS target file.
```

Comment:
```suggestion
    A class to hold information about a TESS target file.
```

Don't need the `data` -- it is no different than other Python classes. The `@dataclass` just lets you avoid a bunch of tedious stuff in defining the class.